test/jdk/java/foreign/TestVarArgs.java

```diff
@@ -65,7 +65,7 @@
         System.loadLibrary("VarArgs");
     }
 
-    static final NativeSymbol VARARGS_ADDR = findNativeOrThrow(TestVarArgs.class, "varargs");
+    static final NativeSymbol VARARGS_ADDR = new NativeTestHelper().findNativeOrThrow("varargs");
```

Comment:
Maybe instead of creating `newNativeTestHelper` you can extend that class (as in other cases)

Reply:
Reverting back to a static method